Content.Client/State/GameScreen.cs

```diff
@@ -27,7 +29,15 @@ public override void Startup()
         {
             base.Startup();
 
-            _gameChat = new ChatBox();
+            if(_groupController.CanCommand("asay"))
+            {
+                _gameChat = new ChatBox(true);
+            }
+            else
+            {
+                _gameChat = new ChatBox(false);
+            }
```

Comment:
Why not just do this inside ChatBox's constructor so you don't have to pass a bool to it?

Reply:
An excellent point.